packages/strapi-plugin-graphql/services/Query.js

```diff
@@ -261,22 +250,20 @@ module.exports = {
           query: {
             value: {
               ...this.convertToParams(_.omit(_options, 'where'), model.primaryKey),
-              ..._options.where,
-              // Avoid population.
-              _populate: model.associations.filter(a => !a.dominant && _.isEmpty(a.model)).map(a => a.alias),
```

Comment:
It was an optimization to avoid useless findOne queries... idk if you transferred the logic anywhere else?